api/src/opentrons/config/advanced_settings.py

```diff
@@ -230,6 +230,19 @@ class Setting(NamedTuple):
         ),
         robot_type=[RobotTypeEnum.FLEX],
     ),
+    SettingDefinition(
+        _id="enableCamera",
+        title="Enable OT-2 or Flex Camera",
+        description="This setting enables the use of the OT-2 or Flex Embedded Camera.",
+        robot_type=[RobotTypeEnum.OT2, RobotTypeEnum.FLEX],
+        default_true_on_robot_types=[RobotTypeEnum.OT2, RobotTypeEnum.FLEX],
+    ),
+    SettingDefinition(
+        _id="enableLiveStream",
+        title="Enable Opentrons Live Stream",
+        description="This setting enabled the Opentrons Live Stream service.",
```

Comment:
```suggestion
        description="This setting enables the Opentrons Flex Live Stream service.",
```
Nit, just to keep the setting descriptions consistent.